terragrunt/aws/cloudfront/certificates.tf

```
@@ -0,0 +1,41 @@
+resource "aws_acm_certificate" "simplify_privacy_statement_certificate" {
+  provider = aws.ca-central-1
```

Here we'll need to use a `us-east-1` provider which we haven't defined yet.  The reason for this is that CloudFront requires its certificates to be in the `us-east-1` region (which is also doubles as their `Global` region in the AWS console).

Here's an example of how you add a new region provider:
https://github.com/cds-snc/url-shortener/blob/642580e461bf96fbfa4f9214e80e2c45930905c0/terragrunt/env/common/provider.tf#L15-L19

If you're curious about how a TF project with multiple providers works, the `provider` defined without an `alias` attribute is considered the default provider and used by all resources that don't explicitly set a `provider` attribute.

```suggestion
  provider = aws.us-east-1
```

---

Is this just a AWS rule?

---

Yup, exactly - some resources just need to be in `us-east-1` and throw cryptic errors if you don't put them there.